src/Parser.php

```diff
@@ -216,25 +216,7 @@ protected function parseFeature()
         while ($this->predictTokenType() !== 'EOS') {
             $node = $this->parseExpression();
 
-            if (is_string($node)) {
-                if ($this->compatibilityMode->shouldRemoveFeatureDescriptionPadding()) {
-                    $text = preg_replace('/^\s{0,' . ($token['indent'] + 2) . '}|\s*$/', '', $node);
-                    $description .= ($description !== null ? "\n" : '') . $text;
-                    continue;
-                }
-
-                if ($node === "\n" && $description === null) {
-                    // Ignore empty lines before the start of the description
-                    continue;
-                }
-
-                // It must be part of the feature description (text & newlines later in the document will be consumed as
-                // part of parsing Background / Scenario before execution returns to this loop).
-                $description .= $node;
-                if ($node !== "\n") {
-                    // Text nodes do not end with a newline, add one. The final trailing newline is rtrimmed below.
-                    $description .= "\n";
-                }
```

Comment:
Any `Text` lines that are validly part of the Feature description will have been consumed by `parseTitleAndDescription` before we enter the loop.

Once in the loop, the only string lines we can get are blank lines - everything else will be consumed (or rejected) as part of processing a `Background` or `Scenario / Scenario Outline` child.